client/app/models/package.js

```diff
@@ -76,6 +76,9 @@ export default class PackageModel extends Model {
     if (this.dcpPackagetype === DCPPACKAGETYPE.RWCDS.code) {
       await this.rwcdsForm.save();
     }
+    if (this.dcpPackagetype === DCPPACKAGETYPE.DRAFT_LU_PACKAGE.code) {
+      await this.landuseForm.save();
+    }
```

Comment:
Can you add landuseForm to `isDirty` below as well?

Reply:
```    if (this.dcpPackagetype === DCPPACKAGETYPE.DRAFT_LU_PACKAGE.code) {
      return isPackageDirty
        || this.landuseForm.hasDirtyAttributes
        || this.landuseForm.isAffectedZoningResolutionsDirty
        || this.landuseForm.isBblsDirty
        || this.landuseForm.isApplicantsDirty
        || this.landuseForm.isLanduseActionsDirty
        || this.landuseForm.isRelatedActionsDirty
    }```

Reply:
Ohh, yes, good catch!! Thank you!! 🙇  🙇  🙇